orion/pkg/repos/user_repo_test.go

```diff
@@ -28,7 +28,7 @@ func TestSelectAllUsers(t *testing.T) {
 	// Mock DB statements and execute
 	rows := getUserRows()
 	mock.ExpectPrepare("^SELECT (.+) FROM users").ExpectQuery().WillReturnRows(rows)
-	got, err := repo.SelectAll()
+	got, err := repo.SelectAll("", 100, 0)
```

Comment:
can we have an additional user_repo_test that tests:
```
mock.ExpectPrepare("^SELECT (.+) FROM users WHERE ___ IN (...)")
```
This should get called when we call repo.SelectAll("something", ...).

Thanks!